data/campaigns/Sceptre_of_Fire/scenarios/8_The_Dragon.cfg

```diff
@@ -118,11 +118,42 @@
     [event]
         name=prestart
 
+        {VARIABLE found_forge no}
+
         [objectives]
             side=1
             [objective]
                 description= _ "Get all heroes to the end of the tunnel"
                 condition=win
+                [show_if]
+                    {VARIABLE_CONDITIONAL found_forge equals no}
```

Comment:
Using boolean_equals is preferred.
`equals` matches for exact strings
`boolean_equals` in difference would as well work if the variable is undefined.